Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -775,7 +775,7 @@
 			repositoryURL = "https://github.com/paypal/paypalcheckout-ios";
 			requirement = {
 				kind = exactVersion;
-				version = 0.112.1;
+				version = 1.0.0;
```

Comment:
@KunJeongPark can you create a follow up ticket to remove `PayPalCheckout` as a package dependency in the demo app? It's used because original we thought we wanted merchants to pull in `PayPalCheckout` directly. Since we have moved away from this we should mock the NXO classes in our demo app vs using their classes directly.

Reply:
Jax, I am not 100% sure what you mean by "mock the NXO classes" in the demo app

Reply:
I am not sure why we need to do this Jax.

Reply:
Merchants shouldn't know about `PayPalCheckout` at all and right now our demo app displays that they should both use our `PayPalNativePayments` _and_  pull in `PayPalCheckout`. This is not a path that we want merchants to take so we should remove it from our demo app. Our demo app should display how we expect merchant to use `PayPalNativePayments`.

Reply:
Jax is right - a merchant should never integrate with PayPalCheckout directly. Only the PPCP SDK should.

We have parts of the demo app that fetch orders via PayPalCheckout, when it should instead use our sample server.

Reply:
Here is the jira ticket to remove PayPalCheckout as Package Dependency in demo app. https://paypal.atlassian.net/browse/DTNOR-1007